src/applications/vaos/services/appointment/index.js

```diff
@@ -677,3 +678,120 @@ export async function cancelAppointment({ appointment }) {
+  const isHome = isVideoHome(appointment);
+  const videoKind = appointment?.videoData.kind;
+  const isVideo = appointment?.vaos.isVideo;
+  const isCommunityCare = appointment?.vaos.isCommunityCare;
+  const isPhone = isVAPhoneAppointment(appointment);
+  const isInPersonVAAppointment = !isVideo && !isCommunityCare && !isPhone;
+
+  if (isPhone) {
+    data = {
+      summary: 'Phone appointment',
+      location: formatFacilityAddress(facility),
+      text: `A provider will call you at ${moment
+        .parseZone(appointment.start)
+        .format('h:mm a')}`,
+      phone: formatFacilityPhone(facility),
+      additionalText: [
+        'Sign in to VA.gov to get details about this appointment',
+      ],
+    };
+  } else if (isInPersonVAAppointment) {
+    data = {
+      summary: `Appointment at ${facility?.name}`,
+      location: formatFacilityAddress(facility),
+      text: `You have a health care appointment at ${facility?.name}`,
+      phone: formatFacilityPhone(facility),
+      additionalText: [
+        'Sign in to VA.gov to get details about this appointment',
+      ],
+    };
+  } else if (isCommunityCare) {
+    const { providerName, practiceName } =
+      appointment.communityCareProvider || {};
+
+    data = {
+      summary: `Appointment at ${providerName || practiceName}`,
+      providerName: `${providerName || practiceName}`,
+      location: `${formatFacilityAddress(appointment.communityCareProvider)}`,
+      text:
+        'You have a health care appointment with a community care provider. Please don’t go to your local VA health facility.',
+      phone: `${formatFacilityPhone(appointment.communityCareProvider)}`,
+      additionalText: [
+        'Sign in to VA.gov to get details about this appointment',
+      ],
+    };
+  } else if (isVideo) {
+    const providerName = appointment.videoData?.providers
+      ? appointment.videoData.providers[0]?.display
+      : '';
+
+    if (isHome) {
+      data = {
+        summary: 'VA Video Connect appointment',
+        text:
+          'You can join this meeting up to 30 minutes before the start time.',
+        location: 'VA Video Connect at home',
+        additionalText: ['Sign in to VA.gov to join this meeting'],
+      };
+    } else if (isAtlas) {
+      const { atlasLocation } = appointment.videoData;
+
+      if (atlasLocation?.address) {
+        data = {
+          // TODO: This doesn't seem right
+          summary: `VA Video Connect appointment at an ATLAS facility`,
+          // providerName: `${firstName} ${lastName}`,
+          location: formatFacilityAddress(atlasLocation),
+          text: 'Join this video meeting from this ATLAS (non-VA) location:',
+          additionalText: [
+            `Your appointment code is ${
+              appointment.videoData.atlasConfirmationCode
+            }. Use this code to find your appointment on the computer at ${providerName}.`,
+            `You'll be meeting with ${providerName}`,
```

Comment:
Should this be modified as well? Ex.

Use this code to find your appointment on the computer at the ATLAS facility
Remove 'You'll be meeting with ${providerName}

Reply:
I think you can leave `You'll be meeting with ${providerName}`, but yeah, I think the text should be `Use this code to find your appointment on the computer at the ATLAS facility`. You might want to check that you have a provider name, though, so we don't output "You'll be meeting with" without a name at the end.

Providers from the video appointment data are the names of the VA doctors/providers associated with the appointment.